examples/react-todos-app/src/App.test.jsx

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { describe, expect, it } from 'vitest';
+import App from './App';
+
+describe('App', () => {
+  it('should display the app title', async () => {
+    render(<App title="React" />);
```

Comment:
To be honest, I copied it from the guide and thought it would be similar to`data-cy-root` in Cypress components. But since it seems to work fine without it and we do not need to query the root anywhere, I removed it.

Reply:
Well, it magically stopped working when I removed it so.... I put it back 😅